yaml/_type/set.ts

```diff
@@ -6,15 +6,11 @@
 import { Type } from "../_type.ts";
 import type { Any } from "../_utils.ts";
 
-const { hasOwn } = Object;
-
 function resolveYamlSet(data: Any): boolean {
   if (data === null) return true;
 
   for (const key in data) {
-    if (hasOwn(data, key)) {
```

Comment:
This `hasOwn` is for avoiding to be affected by Object.prototype extension. If someone extend `Object.prototype`, then unrelated `key` appears here.